godot-core/src/builtin/math/xform.rs

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/// Applying inverse transforms.
```

Comment:
This never includes scaling or translation, right? Might be worth mentioning... :thinking:

Reply:
Sadly it does :upside_down_face: - for Plane https://docs.godotengine.org/en/stable/classes/class_plane.html#class-plane-operator-mul-transform3d.